homeassistant/components/reolink/entity.py

```diff
@@ -0,0 +1,59 @@
+"""Reolink parent entity class."""
+
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .host import ReolinkHost
+
+
+class ReolinkCoordinatorEntity(CoordinatorEntity):
+    """Parent class for Reolink Entities."""
+
+    def __init__(self, hass, config):
+        """Initialize ReolinkCoordinatorEntity."""
+        self._hass = hass
+        coordinator = self._hass.data[DOMAIN][config.entry_id].device_coordinator
```

Comment:
```suggestion
        entry_data: TYPE = self._hass.data[DOMAIN][config.entry_id]
        coordinator = entry_data.device_coordinator
```

TYPE needs to be adjusted